client/header/activity-panel/panels/inbox.js

```diff
@@ -32,8 +32,64 @@ class InboxPanel extends Component {
 		this.props.updateCurrentUserData( userDataFields );
 	}
 
+	renderEmptyCard() {
+		return (
+			<ActivityCard
+				className="woocommerce-empty-review-activity-card"
+				title={ __( 'Your inbox is empty', 'woocommerce-admin' ) }
+				icon={ <Gridicon icon="checkmark" size={ 48 } /> }
+			>
+				{ __(
+					/* eslint-disable max-len */
```

Comment:
You can use concatenation to avoid this. Example: https://github.com/woocommerce/woocommerce-admin/blob/master/client/analytics/settings/config.js#L51

Reply:
Much better, thank you!